### ISSUE TITLE
Where I can find the bucket_id(not the bucket_name)?

### ISSUE THREAD

**Issue by GF-Huang** [influxdata/influxdb-client-python#86]
```python
delete_api = client.delete_api()
delete_api.delete('1970-01-01T00:00:00Z', '2020-04-27T00:00:00Z', '_measurement="daily"', 'finance_data', 'administators')
```

```json
{
    "code":"invalid",
    "message":"id must have a length of 16 bytes"
}
```

**Comment by bednar** [comment]
Hi @GF-Huang,

the `delete_api` should also accept a `bucket_name`. It is a bug... we will fix it ASAP.

As a workaround you could use something like that:

```python
from influxdb_client import DeletePredicateRequest
from influxdb_client import DefaultService

predicate_request = DeletePredicateRequest(start='1970-01-01T00:00:00Z', stop='2020-04-27T00:00:00Z', predicate='_measurement="daily"')
service = DefaultService(self.client.api_client)

service.delete_post(delete_predicate_request=predicate_request, bucket='finance_data', org='administators')

```

Regards

**Comment by bednar** [comment]
Hi @GF-Huang,

the `delete_api` is fixed in [1.7.0 milestone](https://github.com/influxdata/influxdb-client-python/milestone/9).

If you would like to test it then install client via:

```pip install git+https://github.com/influxdata/influxdb-client-python.git@master```

Regards

Regards

**Comment by GF-Huang** [comment]
Got it, thanks.
